app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -217,33 +244,41 @@ public void onFocusChange(View view, boolean focus) {
                             Matches.MatchRow Match = Globals.MatchList.getMatchInfoRow(MatchNum);
                             if (Match != null) {
                                 // MUST CONVERT TO STRING or it crashes with out warning
-                                int[] Teams = Match.getListOfTeams();
-                                // TODO Set "Teams" to the options in the single select dropdown
+                            String[] teams_Str = new String[teams.length];
+                            for (int i = 0; i < teams.length; i++) {
+                                teams_Str[i] = String.valueOf(teams[i]);
+                            }
+                            if (teams_Str[0].equals("0")) {
+                                teams_Str = new String[]{"None"};
```

Comment:
"None" is "user-facing" so should be in strings.xml so we don't have it hard-coded  here.  (i think there was another place where you load the adp_Team and use the string there too.

Reply:
I know I was just quickly writing it and forgot to fix it.